reports/Illinois.md

```diff
@@ -190,3 +187,31 @@ id: il-joliet-1
 * https://www.youtube.com/watch?v=rllO2sdj1Ek
 * https://twitter.com/OrozcoJustina/status/1268262843801010179
 * https://www.chicagotribune.com/news/breaking/ct-joliet-mayor-video-fallout-20200606-zeidd3u5xbadxfoedmfkp22akm-story.html
+
+
+### DOJ files "using internet to incite riot" case against Peoria man | May 31st
+
+The U.S Department of Justice has brought federal charges against a Peoria man, Ca'Quintez Gibson for using the internet to incite a riot, apparently linking several incidents of looting to him through no more than general Facebook posts. If convicted, Gibson faces up to five years in prison, a fine of up to $250,000, and three years of supervised release following imprisonment.
+
+tags: arrest, inhumane-treatment
+
+id: il-peoria-1
+
+**Links**
+
+* https://www.justice.gov/usao-cdil/pr/federal-complaint-charges-peoria-man-using-internet-incite-rioting-peoria
+
+
+## Streamwood
+
+### Police show up at teen's house after posting about protest | June 4th
+
+Police show up at a teen's house to ask him questions about a protest planned for that night. The filmer states that he did not plan the protest, but was reposting it. At the end of the video, the officer begins to ask, "Who -" before the footage is cut off, suggesting the officer was trying to find the protest organizer.
+
+tags: threaten
+
+id: il-streamwood-1
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268649159952936962
```

Comment:
This isn't police brutality.

Reply:
I think this is certainly abuse of power, which is allowed. I can remove it if you want, but this to me absolutely seems like misconduct we're tracking

Reply:
Fair enough, I can see this falling in the same territory as the incident in #626. I think this would be a good one for someone else to cast their eye over, specifically maybe someone else with more first-hand experience of US cops and not a Brit like me!

I know it's a pain, but if you were to remove it from this PR and open it separately I think all the other updates in here could be merged now :+1:

Reply:
Done. I'm going to post this, the Salem, Oregon case and the Fargo case in the discord